cmd/webhook/main.go

```diff
@@ -193,4 +199,7 @@ func main() {
 			webhook.SetCustomizedInjections(cm)
 		}
 	}
+
+	// TODO: find a way to stop cache, should we run the above block in a go routine and make main module
```

Comment:
@martinkennelly @zshi-redhat shall I make to run above for block into a goroutine, exit gracefully on term signal ? this change would also enable to stop nad cache informer.

or is the above for block intentional ?

Reply:
looks this issue is being addressed in this [PR](https://github.com/k8snetworkplumbingwg/network-resources-injector/pull/62), will update it here once that PR is merged.